CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## v43.0.0-SNAPSHOT - unreleased
 
+### 🎁 New Features
+* Added observable `XH.sizingMode` to govern app-wide `sizingMode`. `GridModel`s will bind to this
+  `sizingMode` by default.
+* Added `GridAutosizeMode.ON_SIZING_MODE`, which autosizes Grid columns whenever `GridModel.sizingMode`
```

Comment:
```suggestion
* Added `GridAutosizeMode.ON_SIZING_MODE`, which autosizes Grid columns whenever `XH.sizingMode`
```

Reply:
Actually scratch that - GridModel does publish that var, which is what's actually read.